gm4_area_markers/data/area_markers/functions/marker/process.mcfunction

```diff
@@ -0,0 +1,10 @@
+#called from main
+#@s = area marker
+
+#make message apear
+execute as @s[tag=gm4_am_tier_1] at @s positioned ~-24 0 ~-24 run title @a[dx=48, dy=1000000000000, dz=48] actionbar [{"translate":"%1$s%3427655$s","with":["Area: ",{"translate":"text.gm4.area_markers.area"}],"color":"gold"},{"selector":"@s","color":"white"}]

```

Comment:
This is already running as the AEC, so just put the tag check where you do the `at @s`
```suggestion
execute at @s[tag=gm4_am_tier_1] positioned ~-24 0 ~-24 run title @a[dx=48, dy=1000000000000, dz=48] actionbar [{"translate":"%1$s%3427655$s","with":["Area: ",{"translate":"text.gm4.area_markers.area"}],"color":"gold"},{"selector":"@s","color":"white"}]
```